src/server/service/graphql/schema/mutations/mod.rs

```diff
@@ -188,6 +188,18 @@ impl NotASupplierInvoice {
     }
 }
 
+pub struct CannotDeleteInvoiceWithLines(pub Connector<InvoiceLineNode>);
+#[Object]
+impl CannotDeleteInvoiceWithLines {
+    pub async fn description(&self) -> &'static str {
```

Comment:
The usual comment (will only make it here, the others offenders are easy to find).

```suggestion
    pub async fn description() -> &'static str {
```

I think the `static` might be redundant, too.